packages/fxa-auth-server/lib/error.js

```diff
@@ -1189,6 +1190,15 @@ AppError.insufficientACRValues = foundValue => {
   );
 };
 
+AppError.unknownRefreshToken = () => {
+  return new AppError({
+    code: 400,
+    error: 'Bad Request',
+    errno: ERRNO.REFRESH_TOKEN_UNKNOWN,
+    message: 'Unknown refresh token',
```

Comment:
I haven't added this to the errno list in fxa-content-server, because we swallow it rather than propagating it out to the user.